quantum/process_keycode/process_unicode_common.c

```diff
@@ -64,28 +68,24 @@ void unicode_input_start (void) {
   case UC_LNX:
     register_code(KC_LCTL);
     register_code(KC_LSFT);
-    register_code(KC_U);
-    unregister_code(KC_U);
+    tap_code(KC_U);
     unregister_code(KC_LSFT);
     unregister_code(KC_LCTL);
     break;
   case UC_WIN:
     register_code(KC_LALT);
-    register_code(KC_PPLS);
-    unregister_code(KC_PPLS);
+    tap_code(KC_PPLS);
     break;
   case UC_WINC:
-    register_code(KC_RALT);
-    unregister_code(KC_RALT);
-    register_code(KC_U);
-    unregister_code(KC_U);
+    tap_code(KC_RALT);
+    tap_code(KC_U);
   }
   wait_ms(UNICODE_TYPE_DELAY);
 }
 
 __attribute__((weak))
 void unicode_input_finish (void) {
-  switch(input_mode) {
+  switch(unicode_config.input_mode) {
```

Comment:
```suggestion
  switch (unicode_config.input_mode) {
```
Add space before `(` as it's not a function call.